datafusion/physical-expr/src/aggregate/first_last.rs

```diff
@@ -54,6 +55,24 @@ impl FirstValue {
         input_data_type: DataType,
         ordering_req: LexOrdering,
         order_by_data_types: Vec<DataType>,
+    ) -> Self {
+        Self::with_ignore_null(
+            expr,
+            name,
+            input_data_type,
+            ordering_req,
+            order_by_data_types,
+            false,
+        )
+    }
+
+    pub fn with_ignore_null(
```

Comment:
Instead of this API, maybe we can use builder API as below
```rust
pub fn with_ignore_null(mut self, ignore_null: bool) -> Self{
    self.ignore_null = ignore_null;
    self
}
```
However, this purely stylistic. Feel free to proceed as you wish

Reply:
Changed. Thanks for the suggestion!